src/components/UploadPopup.tsx

```diff
@@ -1,5 +1,45 @@
-function UploadPopup() {
-  return <div></div>
+import UploadOption from './UploadOption'
+import '../styles/UploadVideo.css'
+
+type PopupProp = {
+  trigger: boolean
+  setTrigger: (value: boolean) => void
+}
+
+function UploadPopup(props: PopupProp) {
+  return props.trigger ? (
```

Comment:
might be cleaner to render popup only if trigger is true from outside:
{
    trigger &&
    UploadPopup
}